src/adaptations/device-layer/include/Weave/DeviceLayer/ConfigurationManager.h

```diff
@@ -111,6 +123,11 @@ class ConfigurationManager
     bool IsPairedToAccount();
     bool IsMemberOfFabric();
     bool IsFullyProvisioned();
+    bool DeviceCredentialsProvisioned();
```

Comment:
For clarity, the name of this method should probably include the word "Operational".